sentence.go

```diff
@@ -19,6 +19,56 @@ const (
 	ChecksumSep = "*"
 )
 
+var (
+	customParsers          = map[string]parserCallbackType{}
+	defaultSentenceParsers = map[string]parserCallbackType{
+		TypeRMC: func(s BaseSentence) (Sentence, error) {
+			return newRMC(s)
+		},
+		TypeGGA: func(s BaseSentence) (Sentence, error) {
+			return newGGA(s)
+		},
+		TypeGSA: func(s BaseSentence) (Sentence, error) {
+			return newGSA(s)
+		},
+		TypeGLL: func(s BaseSentence) (Sentence, error) {
+			return newGLL(s)
+		},
+		TypeVTG: func(s BaseSentence) (Sentence, error) {
+			return newVTG(s)
+		},
+		TypeZDA: func(s BaseSentence) (Sentence, error) {
+			return newZDA(s)
+		},
+		TypePGRME: func(s BaseSentence) (Sentence, error) {
+			return newPGRME(s)
+		},
+		TypeGSV: func(s BaseSentence) (Sentence, error) {
+			return newGSV(s)
+		},
+		TypeHDT: func(s BaseSentence) (Sentence, error) {
+			return newHDT(s)
+		},
+		TypeGNS: func(s BaseSentence) (Sentence, error) {
+			return newGNS(s)
+		},
+		TypeTHS: func(s BaseSentence) (Sentence, error) {
+			return newTHS(s)
+		},
+		TypeWPL: func(s BaseSentence) (Sentence, error) {
+			return newWPL(s)
+		},
+		TypeRTE: func(s BaseSentence) (Sentence, error) {
+			return newRTE(s)
+		},
+		TypeVHW: func(s BaseSentence) (Sentence, error) {
+			return newVHW(s)
+		},
+	}
+)
+
+type parserCallbackType func(BaseSentence) (Sentence, error)
```

Comment:
I suggest renaming and exporting `parserCallbackType` to `ParserCallback` and then use it in the `MustRegisterParser` and `RegisterParser` functions.